hanjaelee/app.js

```diff
@@ -52,20 +53,175 @@ app.post('/signup', (req, res) => {
       profile_image,
       password
     ) VALUES (?, ?, ?, ?);`
-  const rawData = database.query(rawQuery, [name, email, profileImage, password])
 
-  rawData
-    .then((data) => {
-      if (data.affectedRows) {
-        res.status(201).json({ message: 'userCreated' })
-        return
-      }
+  try {
+    const rawData = await database.query(rawQuery, [name, email, profileImage, password])
+    if (rawData['affectedRows']) {
+      return res.status(201).json({ message: 'userCreated' })
+    }
+    return res.status(202).json({ message: 'userNotCreated' })
+  } catch (error) {
+    return res.status(400).json({ message: error.sqlMessage })
+  }
+})
```

Comment:
엄청 깔끔하네요 한재님 👍

- 코드 라인을 조금 구분하면 구조 파악에 더 용이할 것 같습니다!!
- 로그에서 에러를 파악하고 디버깅 할 수 있도록 `console.error()` 구문을 추가해주면 좋을 것 같습니다.

```suggestion
  try {
    const rawData = await database.query(rawQuery, [name, email, profileImage, password])
    
    if (rawData['affectedRows']) {
      return res.status(201).json({ message: 'userCreated' })
    }
    
    return res.status(202).json({ message: 'userNotCreated' })
  } catch (error) {
    console.error("Failed to Signup", error)
    return res.status(400).json({ message: error.sqlMessage })
  }
})
```